Fix: provide option to set decimal_places_display explicitly #521 (#555)
django-money/django-money: docs/changes.rst

Changelog
=========

`Unreleased`_
-------------

Added
~~~~~

> - Possibility to set localized string output to length of `decimal_places_display` `#521`_ (`wearebasti`)

Reviewer: ```suggestion
- Improved localization: New setting ``CURRENCY_DECIMAL_PLACES_DISPLAY`` configures decimal places to display for each configured currency `#521`_ (`wearebasti`)
```

Changelog
=========

> `Unreleased`_

Reviewer: ```suggestion
`1.2`_ - TBA
```
This needs to be revered, 1.2 is already a known link, I think the other one will break the docs build.

> -------------

Reviewer: ```suggestion
------------
```
